src/nyc_trees/apps/core/models.py

```diff
@@ -105,6 +105,20 @@ def online_training_complete(self):
     def training_complete(self):
         return self.online_training_complete and self.field_training_complete
 
+    @property
+    def qualified_as_individual_mapper(self):
+        """
+        Return True if user has completed field training and at least
+        one other event.
+        """
+        # Has an admin rescinded mapper status?
+        if self.individual_mapper is False:
+            return False
+        from apps.event.models import EventRegistration
+        return self.field_training_complete and \
+            EventRegistration.objects.filter(user=self,
+                                             did_attend=True).count() > 1
```

Comment:
Although I dislike having to import `EventRegistration` here to avoid a circular-dependency issue, I still prefer doing this rather than using `xxx_set`.